Tests/Endpoints/PermissionTemplatesTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/** @noinspection PhpUnhandledExceptionInspection */
+
+namespace Lokalise\Tests\Endpoints;
+
+use Lokalise\Endpoints\Endpoint;
+use Lokalise\Endpoints\EndpointInterface;
+use Lokalise\Endpoints\PermissionTemplates;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+final class PermissionTemplatesTest extends TestCase
+{
+    use MockEndpointTrait;
+
+    /** @var MockObject|PermissionTemplates */
+    private $mockedPermisionTemplates;
+
+    protected function setUp(): void
+    {
+        $this->mockedContributors = $this->createEndpointMock(PermissionTemplates::class);
```

Comment:
🟢 If you find time and strength to do it, maybe rename for clarity 👍

Reply:
maaan, how did I overlook that 😨